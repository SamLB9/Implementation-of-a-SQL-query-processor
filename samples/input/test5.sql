SELECT * FROM Car, Manufacturer WHERE Car.ManufacturerID = Manufacturer.ManufacturerID;