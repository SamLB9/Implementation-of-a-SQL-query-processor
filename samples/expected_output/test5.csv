1, 1, 2019, 1, 80000, 1, 101, 1001
2, 2, 2020, 1, 40000, 1, 101, 1001
3, 3, 2018, 2, 20000, 2, 102, 1002
4, 3, 2017, 2, 25000, 2, 102, 1002
5, 1, 2019, 3, 18000, 3, 103, 1003